app/lib/bulk_download_service.rb

```diff
@@ -14,17 +14,39 @@ class BulkDownloadService
   #   - +user+ (User) => User requesting download
   #   - +study_bucket_map+ => Map of study IDs to bucket names
   #   - +output_pathname_map+ => Map of study file IDs to output pathnames
+  #   - +host+ => hostname of the server instance, used to generate manifest links
   #
   # * *return*
   #   - (String) => String representation of signed URLs and output filepaths to pass to curl
-  def self.generate_curl_configuration(study_files:, user:, study_bucket_map:, output_pathname_map:)
+  def self.generate_curl_configuration(study_files:,
+                                       user:,
+                                       study_bucket_map:,
+                                       output_pathname_map:,
+                                       hostname:)
     curl_configs = ['--create-dirs', '--compressed']
     # Get signed URLs for all files in the requested download objects, and update user quota
     Parallel.map(study_files, in_threads: 100) do |study_file|
       client = FireCloudClient.new
       curl_configs << self.get_single_curl_command(file: study_file, fc_client: client, user: user,
                                                    study_bucket_map: study_bucket_map, output_pathname_map: output_pathname_map)
     end
+    studies = study_files.map(&:study).uniq
+    study_manifest_paths = studies.map do |study|
+      "#{Rails.application.routes.url_helpers.manifest_api_v1_study_path(study)}"
```

Comment:
since you have the include now, this isn't needed.  Furthermore, you can call `manifest_api_v1_study_url(study)` to get the fully-qualified URL.  If that barks because of a missing host/protocol, then add this:

```
url_params = {id: study.id}.merge(ApplicationController.default_url_options)
manifest_api_v1_study_url(url_params)
```